src/raster/r.forcircular/r.forcircular

```diff
@@ -0,0 +1,2039 @@
+#!/usr/bin/env python3
+# -- coding: utf-8 --
+#
+############################################################################
+#
+# MODULE:      r.forcircular
+# AUTHOR(S):   Francesco Geri, Sandro Sacchelli
+# PURPOSE:     Evaluation of circular bioconomy level of forest ecosystems
+# COPYRIGHT:   (C) 2022 by the GRASS Development Team
+#
+#              This program is free software under the GNU General Public
+#              License (>=v2). Read the file COPYING that comes with GRASS
+#              for details.
+#
+#############################################################################
+#%Module
+#% description: Evaluation of circular bioconomy level of forest ecosystems
+#% keyword: bioeconomy
+#% keyword: forest
+#% overwrite: yes
+#%End
+#%option G_OPT_V_INPUT
+#% key: forest
+#% type: string
+#% description: Name of vector parcel map
+#% label: Name of vector parcel map
+#% required : yes
+#% guisection: Base
+#%end
+#%option G_OPT_V_INPUT
+#% key: boundaries
+#% type: string
+#% description: Name of vector boundaries map (boolean map)
+#% label: Name of vector boundaries map (boolean map)
+#% required : yes
+#% guisection: Base
+#%end
+#%option G_OPT_R_INPUT
+#% key: dtm
+#% type: string
+#% description: Name of Digital terrain model map
```

Comment:
```suggestion
#% description: Name of input elevation raster map
```

Please consider to use predefined phrases (then you get the translation for free):

e.g., `r.slope.aspect --help`
--> "Name of input elevation raster map"

Reply:
There is: G_OPT_R_ELEV (https://grass.osgeo.org/grass80/manuals/parser_standard_options.html)